VERSION

```diff
@@ -1 +0,0 @@
-v1.3
```

Comment:
why are you removing this line?

Reply:
Renamed the file to .version, this is the preferred name for the version file.

Reply:
D'oh, forget to add the new file! Should be there now.